### PR TITLE
add web scraping module

### PR DESCRIPTION
**Pull request by ThomasByr**
scraping of
- https://www.listentech.com/product-category/radio-frequency/150mhz/accessories-150mhz/batteries-150mhz/
- https://secondlifestorage.com/index.php?pages/cell-database/

might need to refactor some stuff and reformat files ... will see

### CODE REVIEW

**Comment by ThomasByr** [comment]
> <picture>
>   <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/Mqxx/GitHub-Markdown/main/blockquotes/badge/light-theme/warning.svg">
>   Warning
> </picture><br>
>
> pretrain module is not made available for now

File: src/webSrap/webScrap.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import argparse
```
Comment (LouiseCouture) [inline src/webSrap/webScrap.py:1]: module pour web scrapping
usage: python webScrap.py --output data.json

File: src/webSrap/batterie/batterie/spiders/bat_spider.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
+import scrapy
```
Comment (LouiseCouture) [inline src/webSrap/batterie/batterie/spiders/bat_spider.py:1]: spiders en charge de crawling les liens pour récupérer les informations des batteries

**Review by LouiseCouture — COMMENTED** [review]
projet Scrapy terminé

File: src/webSrap/webScrap.py (hunk 1 above)
Comment (ThomasByr) [reply src/webSrap/webScrap.py:1]: or
`python chat-bot.py do scrap -o|--output <path-to-file>` 👀

**Review by Bas6700 — APPROVED** [review]

**Pull request merged**